Libraries/Components/Switch/Switch.js

```diff
@@ -147,6 +154,13 @@ export default function Switch(props: Props): React.Node {
   const nativeSwitchRef = React.useRef<?React.ElementRef<
     typeof SwitchNativeComponent | typeof AndroidSwitchNativeComponent,
   >>(null);
+  const _setNativeRef = setAndForwardRef({
```

Comment:
Can this just be useImperativeHandle(nativeSwitchRef)?

Reply:
I don't think this would work since nativeSwitchRef is just the internal ref used by this component. We need to set the ref to both nativeSwitchRef and forwarded ref, which is what setAndForwardRef was made for if I understand correctly.

Reply:
Whoops, my api usage is wrong.

I think this

```

useImperativeHandle(forwardedRef, () => nativeSwitchRef, [nativeSwitchRef])

```

setAndForwardRef is needed if the component attaches a bunch of other methods to the ref too. But that doesn't seem to be the case here

Reply:
Interesting, haven't seen this pattern before, but yea it would work. Seems like it is used here https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/Libraries/Components/Pressable/Pressable.js#L170